internal/translations/catalog.go

```diff
@@ -0,0 +1,68 @@
+// Code generated by running "go generate" in golang.org/x/text. DO NOT EDIT.
```

Comment:
Do we actually check this code in, or is it generated by the build and kept on the local machine only?

Reply:
This code will automatically be generated by go generate command and is responsible for mapping strings to their translations. As we are checking in the translations, I thought of adding the generated catalog.go file as well.